src/app/[locale]/(logged-out)/about/cohort-discovery/components/CtaOverride/CtaOverride.tsx

```diff
@@ -56,8 +56,8 @@ const CtaOverride = ({ ctaLink }: { ctaLink: CtaLink }) => {
     }, [accessData, isClicked, isLoggedIn]);
 
     const isDisabled =
-        (isLoggedIn && (userData === undefined || userData === null)) ||
-        (isLoggedIn && userData?.request_status !== "APPROVED");
+        isLoggedIn && userData ? userData.request_status !== "APPROVED" : false;
```

Comment:
Doesn't the original equate to this? Because in the original, if `userData` is not defined or is null, then it returns true?
```suggestion
        isLoggedIn && userData ? userData.request_status !== "APPROVED" : true;
```

Reply:
Not exactly
We want it false if:
I'm a user who is logged in and I have not made a cohort request, so my userData would be undefined. 
With the new logic it would return false
With the old logic it would return true because of:

```
|| <------------- this lovely or coupled with
        (isLoggedIn && userData?.request_status !== "APPROVED");
```
userData?.request_status !== "APPROVED" is not equal to approved, is undefined because userData is undefined. therefore return true, but we want it false

Reply:
![image](https://github.com/user-attachments/assets/85bcc8ae-97b1-4f07-b061-b3687dbc841d)
<img width="892" alt="image" src="https://github.com/user-attachments/assets/bb2a0eb1-48b0-4cea-9d65-b14b7d831ae4">

Reply:
Great - I hadn't realised the change was desired - I thought you were just simplifying the logic!